incubator/webpagetest-agent/values.yaml

```diff
@@ -0,0 +1,26 @@
+# Default values for webpagetest-agent.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+image:
+  repository: webpagetest/agent
+  tag: latest
```

Comment:
Please use a tagged release instead of latest. Using latest can have a number of cases where problems will arise.

Reply:
@pmeenan Can we have more tags than just `latest`? The `latest` image is 1GB so I'd prefer people did not have to download every time for every pod

Reply:
If you look at the [review guidelines](https://github.com/kubernetes/charts/blob/master/REVIEW_GUIDELINES.md#configuration) we are looking for stable releases. This is important so that multiple installs are repeatable which you can't get with latest. Latest could be one version in a CI run and then when pushed to production later it could have changed. So, we would like tags that don't move for use here.